assets/apps/customizer-controls/src/group-select/GroupSelectComponent.js

```diff
@@ -0,0 +1,83 @@
+import { SelectControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { maybeParseJson } from '@neve-wp/components';
+
+const GroupSelectComponent = ({ control }) => {
+	const { options, label, disabled, description } = control.params;
+	const [value, setValue] = useState(maybeParseJson(control.setting.get()));
+
+	const onChange = (nextVal) => {
+		const saveValue = {};
+		nextVal.forEach((option) => {
+			const currentOpt = JSON.parse(option);
+			if (!saveValue[currentOpt.group]) {
+				saveValue[currentOpt.group] = [];
+			}
+			saveValue[currentOpt.group].push(currentOpt.value);
+		});
+		setValue(saveValue);
+		control.setting.set(JSON.stringify(saveValue));
+	};
+
+	const parsedValue = [];
+	Object.entries(value).forEach((groupOption) => {
+		const groupKey = groupOption[0];
+		groupOption[1].map((val) => {
+			parsedValue.push(JSON.stringify({ group: groupKey, value: val }));
+			return false;
+		});
```

Comment:
Is there any reason to return false from the map? Can't we do a `forEach` instead? 

You can probably also destructure this to improve readability:

```suggestion
         const [ groupKey, groupValue ] = groupOption;

 		groupValue.forEach((val) => {
			parsedValue.push(JSON.stringify({ group: groupKey, value: val }));
		});
```